tests/test_documents.py

```diff
@@ -0,0 +1,142 @@
+                type="class",
+            ),
+            dm.CapellaWorkItem(
+                "ATSY-4321",
+                uuid_capella="2b34c799-769c-42f2-8a1b-4533dba209a0",
+                type="class",
+            ),
+        ]
+    )
+
+    renderer = document_renderer.DocumentRenderer(
+        empty_polarion_worker.polarion_data_repo, model
+    )
+
+    new_doc, wis = renderer.render_document(
+        "jupyter-notebooks/document_templates",
+        "test-classes.html.j2",
+        "_default",
+        "TEST-DOC",
+        cls="c710f1c2-ede6-444e-9e2b-0ff30d7fd040",
+    )
+
+    content = html.fromstring(new_doc.home_page_content.value)
+
+    assert len(wis) == 0
+    assert new_doc.rendering_layouts == [
+        polarion_api.RenderingLayout(
+            label="Class", type="class", layouter="section"
+        )
+    ]
+    assert len(content) == 4
+    assert (
+        etree.tostring(content[0])
+        .decode("utf-8")
+        .startswith("<h1>Class Document</h1>")
+    )
+    assert (
+        etree.tostring(content[2])
+        .decode("utf-8")
+        .startswith(
+            '<div id="polarion_wiki macro name=module-workitem;'
+            'params=id=ATSY-1234|layout=0|external=true"/>'
+        )
+    )
+    assert (
+        etree.tostring(content[3])
+        .decode("utf-8")
+        .startswith(
+            '<div id="polarion_wiki macro name=module-workitem;'
+            'params=id=ATSY-4321|layout=0|external=true"/>'
+        )
+    )
+
+
+def test_update_document(
+    empty_polarion_worker: polarion_worker.CapellaPolarionWorker,
+    model: capellambse.MelodyModel,
+):
+    empty_polarion_worker.polarion_data_repo.update_work_items(
+        [
+            dm.CapellaWorkItem(
+                "ATSY-1234",
+                uuid_capella="c710f1c2-ede6-444e-9e2b-0ff30d7fd040",
+                type="class",
+            ),
+            dm.CapellaWorkItem(
+                "ATSY-4321",
+                uuid_capella="2b34c799-769c-42f2-8a1b-4533dba209a0",
+                type="class",
+            ),
+        ]
+    )
+
+    renderer = document_renderer.DocumentRenderer(
+        empty_polarion_worker.polarion_data_repo, model
+    )
+
+    old_doc = polarion_api.Document(
+        module_folder="_default",
+        module_name="TEST-DOC",
+        rendering_layouts=[
+            polarion_api.RenderingLayout(
+                label="Class",
+                type="class",
+                layouter="section",
+                properties=[{"key": "value"}],
+            )
+        ],
+        home_page_content=polarion_api.TextContent(
+            type="text/html",
+            value='<h2 id="polarion_wiki macro name='
+            'module-workitem;params=id=ATSY-16062"/>',
+        ),
+    )
+
+    new_doc, wis = renderer.render_document(
+        "jupyter-notebooks/document_templates",
+        "test-classes.html.j2",
+        document=old_doc,
+        cls="c710f1c2-ede6-444e-9e2b-0ff30d7fd040",
+    )
```

Comment:
A bit weird to read: This test is about updating a document, but we just `render_document` here. So `remder_document` can do 2 things?